Android/app/src/main/java/com/example/brewhaha_android/Api/BackendInterface.kt

```diff
@@ -19,4 +19,22 @@ interface BackendInterface {
 
     @POST("auth/logout")
     fun logout(@Header("Content-type") contentType: String, @Body user: LogoutUser) : Call<Void>
+
+    @GET("brewery")
+    fun getAllBreweries(@Header("x-access-token") accessToken: String) : Call<List<Brewery>>
+
+    @GET("brewery/{id}")
+    fun getBrewery(@Header("x-access-token") accessToken: String, @Path("id") id: String) : Call<Brewery>
+
+    @PUT("brewery")
+    fun addBrewery(@Header("x-access-token") accessToken: String, @Header("Content-type") contentType: String, @Body brewery: AddBrewery) : Call<Map<String, String>>
+
+    @GET("brewery")
+    fun getBreweryByName(@Header("x-access-token") accessToken: String, @Query("name") breweryName: String) : Call<List<Brewery>>
```

Comment:
I think this can be one endpoint since it's the same path. You can just have three args, and have them be nullable. That will allow us to filter by multiple at once rather than doing many api calls to filter by rating and location and name

Reply:
And in the future, we can just add params to this endpoint. So for reviews and possible family friendly filters

Reply:
Done.